lib/container_server_unsupported.go

```diff
@@ -11,3 +14,8 @@ func (c *ContainerServer) addSandboxPlatform(sb *sandbox.Sandbox) {
 func (c *ContainerServer) removeSandboxPlatform(sb *sandbox.Sandbox) {
 	// nothin' doin'
 }
+
+func (c *ContainerServer) getContainerStats(ctr *oci.Container, previousStats *ContainerStats) (*ContainerStats, error) {
+	// nothin' doin'
+	return nil, error.New("container stats not supported")
```

Comment:
Looks like we need `error` -> [`errors`][1].  And Maybe add that to `import` as well?  Do we have resources for Windows/Solaris CI?  Who is driving the non-Linux CRI-O support?

[1]: https://golang.org/pkg/errors/

Reply:
whups fixed